.github/CODEOWNERS

```diff
@@ -1,3 +1 @@
-* @code-asher @nhooyr
```

Comment:
Not sure if removing CODEOWNERS here was intentional!

Reply:
It was! We strive to break down each PR into individual commits. Makes it much easier to review. See the justification in the second commit.

Creating a new PR for every little change tends to get unwieldy.

Reply:
But yea know what actually, I'll try to do that from now on. It's def confusing to mesh things together in unrelated PRs.